recipes/open62541/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "open62541" %}
+{% set version = "1.1.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/open62541/open62541/archive/v{{ version }}.tar.gz
+    sha256: 29c809334cb28af617d6526b1602b6df07cc33f36fce32f6fc913f8e83564cca
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}              # [not win]
+    - {{ compiler('c') }}                # [not win]
+    - vs2017_win-64                      # [win64]
```

Comment:
This shouldn't be necessary as cf is on vs 2017 by default now

Reply:
Fixed in https://github.com/conda-forge/staged-recipes/pull/13231/commits/11fa452fc9da8680e5e03eebd1d7fc60214f4006 . By the way as this is a C-only library, also the dependency on a C++ compiler was not necessary.

Reply:
Actually as CMake is not properly configured to look just for a C compiler via: 
~~~
project(... LANGUAGES C)
~~~
it uses the default setting (i.e. `C` and `CXX` as a enabled languages) so a C++ compiler is actually necessary.